src/main/java/com/hillel/items_exchange/controller/AdvertisementController.java

```diff
@@ -160,15 +160,13 @@ public AdvertisementDto updateAdvertisement(@Validated(Exist.class) @RequestBody
             @ApiResponse(code = 200, message = "OK"),
             @ApiResponse(code = 400, message = "BAD REQUEST"),
             @ApiResponse(code = 403, message = "FORBIDDEN")})
-    public ResponseEntity<HttpStatus> deleteAdvertisement(@Valid @RequestBody AdvertisementDto dto, Principal principal)
+    public ResponseEntity<HttpStatus> deleteAdvertisement(@Valid @RequestBody Long advertisementId, Principal principal)
```

Comment:
Thanks 4 that! 👍 
But you need to check if Swagger displays that parameter and describe it.

Reply:
Ok